src/Components/Sticky/Sticky.tsx

```diff
@@ -32,17 +33,23 @@ export const Sticky: React.FC<
     withoutHeaderOffset?: boolean
     children: ReactNode | (({ stuck }: { stuck: boolean }) => ReactNode)
   }
-> = ({ children, bottomBoundary, withoutHeaderOffset }) => {
+> = ({
+  children,
+  bottomBoundary,
+  withoutHeaderOffset: _withoutHeaderOffset,
+}) => {
   const { offsetTop, registerSticky, deregisterSticky } = useSticky()
 
   const { desktop, mobile } = useNavBarHeight()
+  const { isEigen } = useSystemContext()
 
   const isMobile = __internal__useMatchMedia(themeProps.mediaQueries.xs)
 
   const [stuck, setStuck] = useState(false)
 
   const containerRef = useRef<HTMLDivElement | null>(null)
 
+  const withoutHeaderOffset = _withoutHeaderOffset ?? isEigen
```

Comment:
or is it better to place this logic in `useNavBarHeight'? Something like this

```javascript
const useNavBarHeight = () => {
  const { isLoggedIn, isEigen } = useSystemContext()

  let mobile = isLoggedIn ? MOBILE_LOGGED_IN_NAV_HEIGHT : MOBILE_NAV_HEIGHT
  let desktop = DESKTOP_NAV_BAR_HEIGHT

  if (isEigen) {
    mobile = 0
    desktop = 0
  }

  return { height: [mobile, desktop], mobile, desktop }
}
```

Reply:
Yeah, this winds up getting used in a few places so it'd make sense to move it to `useNavBarHeight`. I'm surprised it took this long for this to surface!

Reply:
Done